pandas/io/pytables.py

```diff
@@ -348,6 +348,10 @@ def read_hdf(path_or_buf, key=None, **kwargs):
         # so delegate to the iterator
         auto_close = True
 
+    # Remove kwargs which are only relevant for the HDFStore (see GH#16583)
```

Comment:
I would like a narrower change, better to simply name the args here, then they don't need to be passed in.